recipes/ginkgo/all/patches/windows-symbols.patch

```diff
@@ -0,0 +1,710 @@
+diff --git a/core/base/array.cpp b/core/base/array.cpp
+index 0f31b3d9ba7..3a2ad7b5568 100644
```

Comment:
Typically we refrain from back porting features... is this absolutely required?

Reply:
Without it, some MSVC versions won't build shared libraries in some settings due to the number of symbols exceeding 65k. This was only discovered after the release, since for some reason the MSVC in our Github Actions seems to be doing fine. If this patch is too large, I can also download it from Github on-demand.

Reply:
size is fine.. its complexity that usually scares me! since you seem to qwok on the project and it's to enable building more configurations... I think it matches with our unofficial policy #3951